Source/WebKit/UIProcess/mac/WebViewImpl.mm

```diff
@@ -306,8 +306,8 @@ - (void)startObserving:(NSWindow *)window
     NSNotificationCenter *defaultNotificationCenter = [NSNotificationCenter defaultCenter];
 
     // An NSView derived object such as WKView cannot observe these notifications, because NSView itself observes them.
-    [defaultNotificationCenter addObserver:self selector:@selector(_windowDidOrderOffScreen:) name:@"NSWindowDidOrderOffScreenNotification" object:window];
-    [defaultNotificationCenter addObserver:self selector:@selector(_windowDidOrderOnScreen:) name:@"_NSWindowDidBecomeVisible" object:window];
+    [defaultNotificationCenter addObserver:self selector:@selector(_windowDidOrderOffScreen:) name:NSWindowDidOrderOffScreenNotification object:window];
+    [defaultNotificationCenter addObserver:self selector:@selector(_windowDidOrderOnScreen:) name:_NSWindowDidBecomeVisible object:window];
```

Comment:
Do we need `ALLOW_DEPRECATED_DECLARATIONS_BEGIN`/`END` around usages of `_NSWindowDidBecomeVisible` in this file? (It looks like this patch guards `_NSWindowDidBecomeVisible` behind that in WebKitLegacy, but not in WebKit2)

Reply:
Oops, yes. Not sure how I missed that.

Reply:
Even better, we can just use the non-deprecated `NSWindowDidOrderOnScreenNotification`